frontend/src/components/mine/Tailings/MineTailingsFilePicker.js

```diff
@@ -27,15 +27,20 @@ class MineTailingsFilePicker extends Component {
     this.props.fetchMineDocuments(this.props.selectedDocument.mine_guid);
   }
 
-  handleFileSelect = (mineDocumentGuid) => {
-    const data = { mine_document_guid: mineDocumentGuid };
-    this.props
-      .addMineDocumentToExpectedDocument(this.props.selectedDocument.exp_document_guid, data)
-      .then(() => this.refreshUploadedFiles());
+  handleFileSelect = (mine_document_guid) => {
+    const data = { mine_document_guid };
+    this.pushDocument(data);
+  };
+
+  handleFileLoad = (filename, document_manager_guid) => {
+    const data = { filename, document_manager_guid };
```

Comment:
`filename` looks out of place, should it be `fileName` or `file_name` ?

Reply:
I agree it does look a little weird, but the TUS js library uses "filename" so I went with that.

Reply:
I'm used to seeing filename